brianlong/www.validators.app
# 179089436 score v2

#### What's this PR do?
- Creates another pipeline for ValidatorScoreV2, in which scores are based on grouping validators, not medians and averages.

#### How should this be manually tested?
- Let deamons run for a while to collect some data (you'll need all three):
```
rails runner daemons/gather_rpc_mainnet_daemon.rb 
rails runner daemons/validator_score_mainnet_v1_daemon.rb 
rails runner daemons/validator_score_mainnet_v2_daemon.rb
```
- Then do the same for testnet.
- Open home page and switch between V1 and V2 scores. Compare data, make sure scores are assigned correctly. Make sure all old features (like search bar, sorting by stake, sorting by score) still works correctly for both v1 and v2.
Other pages that show validators (like stake pools) should show only V1 scores for now.
- On local machine: try to change NUMBER_OF_GROUPS (eg. to 4), run all 3 deamons again for a while, and see how the scores are assigned now

#### What are the relevant tickets?
- https://www.pivotaltracker.com/n/projects/2177859/stories/179089436

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
